eng/Packages.Data.props

```diff
@@ -12,11 +12,11 @@
   -->
   <ItemGroup Condition="'$(IsClientLibrary)' != 'true'">
     <PackageReference Update="Microsoft.Azure.Amqp" Version="2.4.11" />
-    <PackageReference Update="Microsoft.Azure.Batch" Version="15.0.0" />
+    <PackageReference Update="Microsoft.Azure.Batch" Version="15.1.0" />
     <PackageReference Update="Microsoft.Azure.Devices.Client" Version="1.23.2" />
     <PackageReference Update="Microsoft.Azure.Devices" Version="1.19.0" />
     <PackageReference Update="Microsoft.Azure.KeyVault.Core" Version="3.0.3" />
-    <PackageReference Update="Microsoft.Azure.Management.Batch" Version="4.2.0" />
+    <PackageReference Update="Microsoft.Azure.Management.Batch" Version="14.1.0" />
```

Comment:
14.1.0 or 15.1.0?

Reply:
[14.1](https://github.com/Azure/azure-sdk-for-net/blob/f6710ed0f9327aa2499f04d62d5867c89c162253/sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md). The ControlPlane and DataPlane have different versions.